### PR TITLE
Remove brand repo dependency 💅

### PR DESCRIPTION
**Pull request by robinandeer**
## Describe your changes

Remove brand from ui + onboarding workspaces.
Refactor font preloading.
Replace use of colorsV3 with getColor utility from ui package.
Add UIColor type.
Refactor @font-face and move to global styles definition.
Refactor getColor utility to not lose type safety.
Move font definitions from brand repo to ui package.

## Justify why they are needed

The brand repo is deprecated so we should move stuff to the UI package instead.

I wasn't super careful with the market-web stuff but that should be removed in the future so I think that okey (sorry @gustaveen 🙏)

## Checklist before requesting a review

- [ ] I have performed a self-review of my code
- [ ] Will this be part of a product update? If yes, please write one phrase about this update.

### CODE REVIEW

**Comment by robinandeer** [comment]
Current dependencies on/for this PR:
* main
  * **PR #356** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/356" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/356?utm_source=stack-comment).

File: packages/ui/src/lib/theme/theme.ts [hunk 1]
```diff
@@ -30,6 +30,6 @@ declare module '@emotion/react' {
   export interface Theme extends CustomTheme {}
 }
 
-export const getColor = (color?: string) => {
-  return theme.colors[color as keyof typeof theme.colors] || 'currentColor'
+export const getColor = (color?: UIColor) => {
```
Comment (alebedev) [inline packages/ui/src/lib/theme/theme.ts:33]: Is passing in undefined a valid usage?  We can make code easier to debug if that would be TS error

File: packages/ui/src/lib/theme/theme.test.ts [hunk 2]
```diff
@@ -1,17 +1,18 @@
...
+      expect(getColor('gray100')).toEqual(colors.gray100)
     })
 
     it('should return a simplified color when found in simplifiedColors', () => {
-      expect(getColor('lavender')).toEqual(colorsV3.purple500)
+      expect(getColor('lavender')).toEqual(colors.purple500)
     })
 
     it('should default to currentColor when no other colors can be found', () => {
+      // @ts-ignore - this is a test
```
Comment (alebedev) [inline packages/ui/src/lib/theme/theme.test.ts:15]: @ts-expect-error would be better - unlike @ts-ignore, you'll get notified if line below no longer generates error.  This ensures we only keep valid error ignore comments

**Review by alebedev — APPROVED** [review]
Looks great!

File: packages/ui/src/lib/theme/theme.test.ts (hunk 2 above)
Comment (robinandeer) [reply packages/ui/src/lib/theme/theme.test.ts:15]: [cool](https://app.graphite.dev/macros/HedvigInsurance)

**Review by robinandeer — COMMENTED** [review]
@alebedev I removed the fallback to `currentColor` and the test for it.

I think it's not really expected behaviour and like you said - this makes for a clear API

**Pull request merged**